src/ipvs/ip_vs_conn.c

```diff
@@ -98,7 +98,6 @@ static struct dp_vs_conn *dp_vs_conn_alloc(enum dpvs_fwd_mode fwdmode,
         return NULL;
     }
 
-    memset(conn, 0, sizeof(struct dp_vs_conn));
```

Comment:
Is it safe to use uninitialized dp_vs_conn? Did you examine every field of dp_vs_conn and check whether it may be used uninitialized?

Reply:
It has been tested and no issue is found.

Reply:
I don't think uninitialized dp_vs_conn is good idea. Someday add some field in dp_vs_conn, it may lead problem.

For uninitialized dp_vs_conn maybe we can do something in `dp_vs_conn_free` before invoke `rte_mempool_put`?